src/estimation/include/iDynTree/Estimation/SimpleLeggedOdometry.h

```diff
@@ -233,6 +233,24 @@ class SimpleLeggedOdometry
      */
     bool changeFixedFrame(const FrameIndex newFixedFrame);
 
+    /**
+     * Change the link that the odometry assumes to be fixed
+     * with respect to the inertial/world frame.
+     *
+     * \note The position of the external frame is set by the user
+     */
+    bool changeFixedFrame(const std::string & newFixedFrame,
+                          const Transform & world_H_newFixedFrame);
+
+    /**
+     * Change the link that the odometry assumes to be fixed
+     * with respect to the inertial/world frame.
+     *
+     * \note The position of the external frame is set by the user
+     */
+    bool changeFixedFrame(const FrameIndex newFixedFrame,
```

Comment:
``const`` is not necessary here.

Reply:
But this ensure that in the implementation of the changeFixedFrame we do not accidentally write on `newFixedFrame`, that will not have any effect.

Reply:
but if it is a plain type you cannot write on it. Isn't it a `typedef` to `int`?

Reply:
If I have to tell the truth, I used the `const` identifier because it is widely used in the `SimpleLeggedOdometry` class even if the arguments of the functions are passed by copy.   
However, if you want I can either pass `newFixedFrame` as a `const` reference or pass it as a copy (and as a consequence remove `const`)

Reply:
> but if it is a plain type you cannot write on it. Isn't it a typedef to int?

You can write to the local copy passed to the method. 
Not a big deal, but the rationale is that if you have a method called `bool something(int readyOnly)` method, in the `something` implementation you can write on the `readOnly` variable and the code will compile fine, and that is probably an overlook (perhaps the implementer thinks that `readOnly` is actually a reference, while actually writing on it has no effect outside of the function). If you specify `bool something(const int readyOnly)`, then if you accidentally write on `readOnly` the compilation will fail.